src/query/src/sql.rs

```diff
@@ -380,6 +487,71 @@ mod test {
         )
     }
 
+    #[test]
+    fn test_show_create_table() -> Result<()> {
+        let catalog_name = DEFAULT_CATALOG_NAME;
+        let schema_name = DEFAULT_SCHEMA_NAME;
+        let table_name = "test_table";
+        let schema = vec![
+            ColumnSchema::new(TABLE_COLUMN, ConcreteDataType::string_datatype(), false),
+            ColumnSchema::new(
+                CREATE_TABLE_COLUMN,
+                ConcreteDataType::string_datatype(),
+                false,
+            ),
+        ];
+        let data = vec![
+            Arc::new(StringVector::from(vec!["a".to_string()])) as _,
+            Arc::new(StringVector::from(vec!["table a".to_string()])) as _,
+        ];
+        let expected_columns = vec![
+            Arc::new(StringVector::from(vec!["test_table".to_string()])) as _,
+            Arc::new(StringVector::from(vec!["CREATE TABLE test_table (  table String  , create_table String engine=mock with(regions=1);".to_string()])) as _,
```

Comment:
Can we remove those redundant spaces? Like `test_table (  table` and `table String  ,`.